src/sql_builder/mod.rs

```diff
@@ -129,22 +129,69 @@ impl Bind for &str {
     }
 }
 
-#[test]
-fn it_builds_sql() {
-    #[allow(dead_code)]
+impl<'a, T: Bind> Sealed for &'a [T] {}
+
+impl<'a, T: Bind> Bind for &'a [T] {
+    #[inline]
+    fn reserve(&self) -> usize {
+        let commas_count = self.len().saturating_sub(1);
+        self.iter().map(Bind::reserve).sum::<usize>() + commas_count + 2
+    }
+
+    #[inline]
+    fn write(&self, mut dst: impl fmt::Write) -> fmt::Result {
+        write!(&mut dst, "[")?;
```

Comment:
I suppose it's done in order to support empty lists, innit?

JFYI: It's also possible to use tuples here and pass `(NULL)` for empty lists.
It relies on the property that `NULL` cannot be checked using equality/IN, so it's safe for `Nullable`. On the other hand, it's valid syntax, because of the type propagation.

Reply:
Done